app/assets/javascripts/slide_button_toggle.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
```

Comment:
So I changed it to check to see if 'oregondigital:' is in the path, which should mean that the id exists. I don't even know why I didn't check that the id existed in the first place. :+1: